src/verifier.cc

```diff
@@ -647,8 +647,10 @@ Result Verifier::ProbeSSBO(const ProbeSSBOCommand* command,
                                 "while expected data is not");
   }
 
-  const auto& datum_type = command->GetDatumType();
-  size_t bytes_per_elem = datum_type.ElementSizeInBytes();
+  auto* fmt = command->GetFormat();
```

Comment:
For the sake of clarity, maybe rename this variable to "format" ?

But I can see "fmt" used in other places... Also, beyond this PR, both "cmd" and "command" are used for variable names. Should we be strict on using explicit variable names?

Reply:
I've been going with fmt mostly just because it's shorter to type. I also typically do cmd for command, although sometimes it got typed out fully. I'm not usually too strict on variable naming, I think this shouldn't be to confusing.